pages/events.js

```diff
@@ -30,7 +29,54 @@ export default function Events({ events }) {
       event.date.cfpClose ? new Date(event.date.cfpClose) > new Date() : false
     ),
   };
+  const filters = [
+    {
+      title: "Show all",
+      description: "List all events with no filters",
+      key: "all",
+      icon: FaListUl,
+      current: true,
+    },
+    {
+      title: "CFP open",
+      description: "You can submit a talk to this conference",
+      key: "cfpOpen",
+      icon: FaMicrophoneAlt,
+      current: false,
+    },
+    {
+      title: "In person",
+      description: "These are in person events",
+      key: "inPerson",
+      icon: MdOutlinePeople,
+      current: false,
+    },
+    {
+      title: "Virtual",
+      description: "Held virtually online event",
+      key: "virtual",
+      icon: MdOutlineOnlinePrediction,
+      current: false,
+    },
+  ];
+
+  let displayTabs = filters.flatMap((tab) => {
+    if (tab.key === "all") {
+      return { ...tab, total: categorisedEvents.all.length };
```

Comment:
This could be done directly in the `filters` object. the `if` statement seems unnecessary.

example 

```js
    {
      title: "Show all",
      description: "List all events with no filters",
      key: "all",
      icon: FaListUl,
      current: true,
      total: categorisedEvents.all.length
    }

 ```
and so on

Reply:
This works as good, a good one 👍
![image](https://user-images.githubusercontent.com/95094057/211857432-5b3e0b03-46c9-45ec-aed6-e085baf67e83.png)